e2e_tests/test_record.py

```diff
@@ -169,6 +169,44 @@ def test_record_standard_users_without_perms_cant_see_download_button(
 
         expect(button).to_be_hidden()
 
+    def test_record_alert_banner_is_visible_with_unsupported_file_extension(
```

Comment:
I've been thinking about how we can keep e2e tests minimal to just critical paths and for things that rely upon external api which we mock at the lower level.

I know we are mocking the s3 bucket in the lower level tests but we do have other e2e tests which show that the integration of s3 works correctly for download, for example, so I feel like the logic we have added here is sufficiently tested at the lower level.

What do you think?

Reply:
I agree 👍